roles/create-vms/templates/vm-template.xml.j2

```diff
@@ -3,7 +3,7 @@
   <memory unit='MB'>{{ item.memory }}</memory>
   <vcpu>{{ item.cpus }}</vcpu>  
   <os>
-    <type arch="x86_64" machine="pc-i440fx-4.1">hvm</type>
```

Comment:
this might be fine as I copied the setting from somewhere, but can you explain what your change does.

Reply:
Qemu docs say it is best to not specify machine version, plus it doesn't work on RHEL8 CSB build.